server/routes.js

```diff
@@ -1,7 +1,13 @@
 const router = require('express').Router();
 
 const { slots } = require('./controllers/slots');
+<<<<<<< HEAD
+const {
+  getUser, getLeaderboard, getCountry, getGlobalChat, postGlobalChat,
+} = require('../database/controllers');
+=======
 const roulette = require('./controllers/roulettecontrollers.js');
+>>>>>>> 1964d9395a0110862d0def478b049cd2432ca9f8
```

Comment:
unresolved merge conflict? Or do we keep everything

Reply:
weird. that unresolved conflict didn't notify me.